x-pack/plugins/osquery/public/packs/queries/ecs_mapping_editor_field.tsx

```diff
@@ -230,37 +224,37 @@ const ECSComboboxFieldComponent: React.FC<ECSComboboxFieldProps> = ({
   }, [selectedOptions]);
 
   const availableECSSchemaOptions = useMemo(() => {
-    const currentFormECSFieldValues = map(formData.ecs_mapping, 'key');
+    const currentFormECSFieldValues = map(watchedEcsMapping, 'key');
 
     return ECSSchemaOptions.filter(({ label }) => !currentFormECSFieldValues.includes(label));
-  }, [formData.ecs_mapping]);
+  }, [watchedEcsMapping]);
 
+  // raczej nie ok
   useEffect(() => {
     // @ts-expect-error update types
     setSelected(() => {
-      if (!field.value.length) return [];
+      if (!value?.length) return [];
 
-      const selectedOption = find(ECSSchemaOptions, ['label', field.value]);
+      const selectedOption = find(ECSSchemaOptions, ['label', value]);
 
       return selectedOption
         ? [selectedOption]
         : [
             {
-              label: field.value,
+              label: value,
               value: {
-                value: field.value,
+                value,
               },
             },
           ];
     });
-  }, [field.value]);
+  }, [value]);
 
   return (
     <EuiFormRow
-      label={field.label}
```

Comment:
don't we need `label` anymore?

Reply:
If I remember correctly while I was testing this - the UI is still the same, that label must have been not used or something. 
current - ![Zrzut ekranu 2022-08-22 o 12 01 39](https://user-images.githubusercontent.com/16632552/185895255-2c123b4d-1423-4fc3-b17e-b61812175465.png)

Reply:
ok apparently we needed it, I brought it back in the detection engine pr. and removed the redundant label from outside.